sample/src/main/java/com/schibsted/spain/barista/sample/DatePickerActivity.java

```diff
@@ -2,12 +2,16 @@
 
 import android.app.DatePickerDialog;
 import android.app.Dialog;
+import android.app.TimePickerDialog;
 import android.os.Bundle;
+import android.support.annotation.NonNull;
 import android.support.v4.app.DialogFragment;
 import android.support.v7.app.AppCompatActivity;
 import android.view.View;
 import android.widget.DatePicker;
 import android.widget.TextView;
+import android.widget.TimePicker;
+
 import java.util.Calendar;
 
 public class DatePickerActivity extends AppCompatActivity {
```

Comment:
Change this name. It shouldn't be `DatePickerActivity` anymore, because now it will also have a `TimePicker`. What about `PickersActivity`? Oh! And change also the layout's name. `activity_pickers`?